test/acceptance/content-recommendation.spec.js

```diff
@@ -458,7 +459,7 @@ describe('the content recommendation component', function () {
   it('should set the colors specified to the content recommendation modal', function (done) {
     window.lio = {
       account: {
-        id: 0
+        id: '123'
```

Comment:
why did you change these?

Reply:
because 0 is an invalid account

Reply:
ooooh yah good call. did not realize that was the default. will update.